teams/log.toml

```diff
@@ -0,0 +1,21 @@
+name = "logs"
+subteam-of = "libs"
+
+[people]
+leads = []
+members = [
+    "KodrAus",
+    "sfackler",
+    "Thomasdezeeuw",
+]
+
+[permissions]
+bors.libc.review = true
```

Comment:
I don't think we need this, copy/paste error?

Reply:
Yes, will fix when/if the team is approved.